python-threatexchange/threatexchange/cli/config_cmd.py

```diff
@@ -455,7 +455,7 @@ def execute(self, settings: CLISettings) -> None:
         self.action(settings)
 
     def execute_list(self, settings: CLISettings) -> None:
-        signals = settings.get_all_signal_types(default_to_sample=False)
+        signals = settings.get_all_signal_types()
```

Comment:
With me merging in #1054, this may cause a problem. Apologies.

Reply:
No, I kept the commits separate, I can use rebase -i to only get my changes, you did the right thing!